sw/device/tests/rv_timer_test.c

```diff
@@ -4,24 +4,22 @@
 
 #include "sw/device/lib/rv_timer.h"
 
-#include "sw/device/lib/arch/device.h"
-#include "sw/device/lib/common.h"
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "sw/device/lib/base/log.h"
 #include "sw/device/lib/dif/dif_gpio.h"
 #include "sw/device/lib/irq.h"
 #include "sw/device/lib/pinmux.h"
-#include "sw/device/lib/uart.h"
-
-#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "sw/device/lib/testing/test_main.h"
 
 static dif_gpio_t gpio;
-static uint32_t intr_handling_success = 0;
-static const uint32_t hart = 0;
+// Flag to indicate that the interrupt was seen. Declared as volatile since it
+// is referenced in the ISR routine as well as in the main program flow.
+static volatile bool intr_handling_success;
```

Comment:
Just pointing out that this variable is not initialized here to avoid any surprises once #2160 is addressed and `main()` functions are cleaned.